BUILD

```diff
@@ -50,6 +50,7 @@ filegroup(
     srcs = glob(
         [".git/**"],
         exclude = [".git/**/*[*"],  # gitk creates temp files with []
+        allow_empty = True,
```

Comment:
???? Why do we even have this filegroup.
AFAICT, it is not used in Bazel.
Have you tried the PR without this change?

OTOH, I don't think your change is incorrect, presuming we do need it.
Perhaps it is fine to keep the change in place and to a targeted PR to just remove that target. Let's not mingle the different issues.

Reply:
I tried without the change, and it was failing if I remember right. In general I followed the approach of flipping the flag, and go test by test that was failing and only add the allow_empty where it was needed.
I did not try to remove the filegroup completely.